game/script-snap.rpy

```diff
@@ -0,0 +1,778 @@
+# General tracking; the player unlocks Snap by admitting boredom to Natsuki at least once
+default persistent.jn_snap_unlocked = False
+default persistent.jn_snap_explanation_given = False
+
+# Natsuki will refuse to play with a cheater
+default persistent.jn_snap_player_is_cheater = False
+
+# Transition for the "Snap"! popup
+define popup_hide_transition = Dissolve(0.75)
+
+init 0 python in jn_snap:
+    import random
+    import store
+    import time
+
+    # Card config
+    _card_values = range(1, 11)
+    _card_suits = [
+        "clubs",
+        "diamonds",
+        "hearts",
+        "spades"
+    ]
+
+    _current_table_card_image = "mod_assets/games/snap/cards/blank.png"
+    _turn_indicator_image = "mod_assets/games/snap/ui/turn_indicator_none.png"
+    _snap_popup_image = "mod_assets/games/snap/ui/snap_a.png"
+
+    _SNAP_UI_Z_INDEX = 4
+    _SNAP_POPUP_Z_INDEX = 5
+
+    # Quips
+    _player_correct_snap_quips = [
```

Comment:
Looking at this and other files, we seem to have very inconsistent const styles. Seeing as these shouldn't be changing during runtime, we should use all caps for their names like other consts. And we should probably extend this to the rest of the project

Reply:
Keeping `_current_table_card_image`, `_turn_indicator_image`, `_snap_popup_image ` lowercase as we swap these values constantly throughout the course of a game; otherwise done.